htdocs/product/stock/list.php

```diff
@@ -104,10 +104,14 @@
 $warehouse=new Entrepot($db);
 
 $sql = "SELECT e.rowid, e.ref, e.statut, e.lieu, e.address, e.zip, e.town, e.fk_pays, e.fk_parent,";
-$sql.= " SUM(p.pmp * ps.reel) as estimatedvalue, SUM(p.price * ps.reel) as sellvalue, SUM(ps.reel) as stockqty,";
+$sql.= " SUM(p.pmp * ps.reel) as estimatedvalue, SUM(p.price * ps.reel) as sellvalue, SUM(ps.reel) as stockqty";
 // Add fields from extrafields
-if (! empty($extrafields->attributes[$object->table_element]['label']))
-	foreach ($extrafields->attributes[$object->table_element]['label'] as $key => $val) $sql.=($extrafields->attributes[$object->table_element]['type'][$key] != 'separate' ? "ef.".$key.' as options_'.$key.', ' : '');
+if (! empty($extrafields->attributes[$object->table_element]['label'])){
+	foreach ($extrafields->attributes[$object->table_element]['label'] as $key => $val){
+        $sql.=($extrafields->attributes[$object->table_element]['type'][$key] != 'separate' ? sprintf(", ef.%s as options_%s", $key, $key): '');
```

Comment:
Can you avoid using sprintf and used instead the php contact char ".".
This is more coherent with existing code. also sprintf brings difficult to read code. 
It is just a point of view but we never use this way of coding in existing code, so we must avoid it.